packages/pkgsrc.rb

```diff
@@ -3,31 +3,19 @@
 class Pkgsrc < Package
   description 'pkgsrc is a framework for building third-party software on NetBSD and other UNIX-like systems'
   homepage 'https://github.com/NetBSD/pkgsrc'
-  version '2020Q1'
+  version '2020Q1-1'
```

Comment:
Why not go ahead and upgrade to 2021Q1?  See https://github.com/NetBSD/pkgsrc/tree/pkgsrc-2021Q1.

Reply:
I do wanna get this merged, but we can upgrade it right after.

Reply:
updated